doc-content/enterprise-only/fuse/installing-on-fuse-eap-proc.adoc

```diff
@@ -1,18 +1,20 @@
 [id='installing-on-fuse-eap-proc']
 = Installing {FUSE} on {EAP_LONG}
-Install {FUSE_LONG} 7.6 on {EAP} 7.2.3 to use with {PRODUCT} to deploy integrated services where required.
+Install {FUSE_LONG} {FUSE_VERSION} on {EAP} 7.2.7 to use with {PRODUCT} to deploy integrated services where required.
 
 [NOTE]
 ====
-{FUSE_LONG} 7.6 is supported on {EAP} 7.2.3, whereas {PRODUCT} is supported on {EAP} {EAP_VERSION}. To ensure a stable deployment environment, install {FUSE} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
+{FUSE_LONG} 7.7 is supported on {EAP} 7.2, whereas {PRODUCT} 7.8.x and newer is supported on Red Hat {EAP} 7.3. To ensure a stable deployment environment, install {FUSE}} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
```

Comment:
I'd suggest changing the EAP_VERSION attribute value. 

```suggestion
{FUSE_LONG} {FUSE_VERSION} is supported on {EAP} {EAP_VERSION}, whereas {PRODUCT} 7.8.x and newer is supported on Red Hat {EAP} 7.3. To ensure a stable deployment environment, install {FUSE} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
```

Reply:
Thanks for catching the stray bracket. The reason for not using version attributes here is that Fuse 7.7 only supports EAP 7.2.x, while Fuse 7.8 will support EAP 7.3.x. RHPAM/RHDM 7.9, on release, will be certified with Fuse 7.7 and will contain this Fuse note. RHPAM/RHDM 7.9 will be re-certified with Fuse 7.8 and the Fuse note will be removed.